### ISSUE TITLE
Missing U+3000 IDEOGRAPHIC SPACE

### ISSUE THREAD

**Issue by omixann** [ClintGoss/Kurinto#12]
I've noticed that this particular character is missing from my Kurinto Seri, and as far as I can tell after my sloppy test, it is also missing from every other font.

**Comment by ClintGoss** [comment]
U+3000 is in Kurinto Seri HK, JP, KR, SC, and TC ... but not in Kurinto Seri Aux, CJK, Core, KM, (Regular), and TB.

Are you suggesting that U+3000 is needed in one or more of the variants that currently do not have the character (like CJK, where it might be appropriate)?

**Comment by omixann** [comment]
Sorry, maybe I didn't understand how fonts are structured. I'm using `Fonts/KurintoSans-Rg.ttf`, which has kana but no ideographic space (as a test, I wrote some test using Windows IME for Japanese and characters could be rendered except for that space).

I feel like it's weird to be able to type kana without being able to type the space. IMHO it should definitely be part of CJK, and also part of the font file I mentioned, as it already contains kana.

Also, I noticed 「。」 and 「、」 are missing too. Once again, it surprises me that these characters are missing.

**Comment by ClintGoss** [comment]
Yes, this does look like a significant bug. The [CJK Symbols and Punctuation] block U+3000-303F should be included in the "Main" variant ... Thanks for finding this!!
